forms/FieldList.php

```diff
@@ -163,7 +164,10 @@ public function removeFieldsFromTab($tabName, $fields) {
 		$this->flushFieldsCache();
 
 		// Find the tab
-		$tab = $this->findOrMakeTab($tabName);
+		$tab = $this->findTab($tabName);
+		
+		if (!$tab)
```

Comment:
Please don't return false here. Can you just wrap the foreach in a positive assertion on `$tab`?

eg: `if ($tab) foreach($fields as $field) $tab->removeByName($field);`